src/frontend/screens/Settings/components/EnableEsync.tsx

```diff
@@ -4,12 +4,17 @@ import { ToggleSwitch } from 'frontend/components/UI'
 import useSetting from 'frontend/hooks/useSetting'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+import { isWindows } from 'backend/constants'
 
 const EnableEsync = () => {
   const { t } = useTranslation()
 
   const [enableEsync, setEnableEsync] = useSetting('enableEsync', false)
 
+  if (isWindows) {
```

Comment:
in the other code in the GameSettings we only show this config if linux and non native, we should do the same check here (like we do for fsync)